jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -6602,6 +6602,13 @@ private static char convEndian(boolean isBigEndian, char value) {
 	 */
 	public native boolean isFlattened(Field field);
 
+	/**
+	 * Determines the size of an object in bytes
+	 *
+	 * @param o the object to determine the size of
+	 */
+	public native long getObjectSize(Object o);
```

Comment:
This doesn't seem like the right place for this addition: I don't see it in any version of the RI.

Reply:
Its not in the main repo, but it is in the Valhalla feature repo

Reply:
Then perhaps it should be conditional?
```
/*[IF INLINE-TYPES]*/
```

Reply:
it is already inside of a conditional `/*[IF INLINE-TYPES]*/` block that begins on line 6550 and ends on line 6847

Reply:
Excellent. My bad for not looking at the broader context. Thanks.